src/main/java/com/dubreuia/ui/java/IdeSupportPanel.java

```diff
@@ -35,10 +35,8 @@
 import com.intellij.ui.components.JBLabel;
 import org.jetbrains.annotations.NotNull;
 
-import javax.swing.JButton;
-import javax.swing.JPanel;
-import java.awt.BorderLayout;
-import java.awt.Dimension;
+import javax.swing.*;
```

Comment:
I think you should keep these imports instead of use *

Reply:
Agree. I used the code style attached to this project.

<img width="1537" alt="image" src="https://user-images.githubusercontent.com/4817889/144636402-3d4e1a5f-a29d-4233-9bab-35f3c57ed808.png">

Reply:
Should be fixed by the last commit.